lib/membrane_rtc_engine/endpoints/hls/custom_layout_maker.ex

```diff
@@ -37,18 +11,32 @@ if Code.ensure_loaded?(Membrane.VideoCompositor) do
     alias Membrane.VideoCompositor.VideoTransformations.TextureTransformations.CornersRounding
     alias Membrane.VideoCompositor.VideoTransformations.TextureTransformations.Cropping
 
+    @type track_no :: pos_integer()
+    @type padding :: pos_integer()
```

Comment:
Why `pos_integer` instead of `non_neg_integer`, don't you iterate tracks from 0?